artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java

```diff
@@ -619,6 +649,10 @@ public void setUseLocalTx(final Boolean localTx) {
       this.localTx = localTx;
    }
 
+   public Boolean getRebalanceConnections() {
```

Comment:
why? same question as before.

Reply:
Unfortunately, these methods are not recognized in some environments, causing failure to expose the properties (Websphere Application Server is one such).